src/ray/util/process.h

```diff
@@ -40,6 +40,18 @@ enum { PID_MAX_LIMIT = 1 << 22 };
 
 namespace ray {
 
+#if !defined(_WIN32)
+/// Sets the flag FD_CLOEXEC to a file descriptor.
+/// This means when the process is forked, this fd would be closed in the child process
+/// side. Raylet forks to create core workers and we don't want the Unix Socket FDs to be
+/// inherited by the core workers. Leaking these FDs would have performance implications.
```

Comment:
this section of the comment is an abstraction leak, should be put at the callsite to close those FDs instead